app/controllers/application_controller.rb

```diff
@@ -4,10 +4,18 @@ class ApplicationController < ActionController::Base
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   protect_from_forgery with: :exception
 
-  before_action :authenticate_user!
+  before_action :authenticate_user!, :banned?
```

Comment:
so you will call this method everytime?
maybe you need to check if banned only when user want's to create issue?